CHANGELOG.md

```diff
@@ -5,6 +5,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.9.4] - 2025-04-25
+
+### Added
+
+- Honor XDG_CONFIG_HOME on macOS by @tessus in [#185](https://github.com/orhun/rustypaste-cli/pull/185)
```

Comment:
We need to add an explanation about this.

I will get to it after the current PRs are merged.

Reply:
I forgot to update the README in PR #185. I thought I did, which is why I didn't elaborate further.
I'll create a separate PR with an update to the README.

Reply:
maybe something like this:

```suggestion
- Honor XDG_CONFIG_HOME on macOS (additionally search for the config file at the same locations as on Linux) by @tessus in [#185](https://github.com/orhun/rustypaste-cli/pull/185)
```

Reply:
Is that ^^^ better, or do you want to change it yourself?

Reply:
ah yeh, that should be good.

Feel free to create a new release

Reply:
will do. Sorry that it took so long. I couldn't sleep all night and I finally fell asleep in the morning.